build-system/pr-check.js

```diff
@@ -37,12 +37,11 @@ function timedExecOrDie(cmd) {
 function main() {
   // TODO(danielrozenberg): conditional test running
   timedExecOrDie('eslint .');
-  timedExecOrDie('cd bundle-size && npm ci');
-  timedExecOrDie('cd bundle-size && npm test -u');
-  timedExecOrDie('cd test-status && npm ci');
-  timedExecOrDie('cd test-status && npm test -u');
-  timedExecOrDie('cd pr-deploy && npm ci');
-  timedExecOrDie('cd pr-deploy && npm test -u');
+  ['bundle-size', 'test-status', 'pr-deploy', 'owners'].forEach(appName => {
```

Comment:
Ok to refactor for now but per danielrozenberg's TODO above, this will have to be split out again.

Reply:
What does his TODO mean/refer to?

Reply:
My guess - run only the tests relevant to the PR, similar to how we do it in amphtml.

Reply:
Ah gotcha. Is there a tracking bug for this yet afayk?